client/cli/src/main.rs

```diff
@@ -72,6 +72,7 @@ enum CliOutput {
     },
 
     Info {
+        network: String,
```

Comment:
nit: I think this could be of type `Network`

Reply:
Will piggy back this in next PR. this one's already merged.